### PR TITLE
LIT7068 69

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Two qenes from Abb 145. LIT7069ItyopyaAbbay "Ityopya..." seems to be known and included into an Ethiopian printed collection of qenes, still to be found (perhaps it is in the sizable three volume "Ya-geez qeneyat"). Some excerpts from the prints, including the qene in question, can be found in GitHUb:  `https://github.com/geezorg/ebooks/blob/master/amharic/poetry/Miscellaneous/src/GeezQineyat`. Should it be included this source into bibliography?

### CODE REVIEW

**Review by CarstenHoffmannMarburg — CHANGES_REQUESTED** [review]

**Comment by CarstenHoffmannMarburg** [comment]
> Some excerpts from the prints, including the qene in question, can be found in GitHUb: `https://github.com/geezorg/ebooks/blob/master/amharic/poetry/Miscellaneous/src/GeezQineyat`. Should it be included this source into bibliography?

If it is this text, it should be included.

File: new/LIT7068Emazatekat.xml [hunk 1]
```diff
@@ -0,0 +1,81 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7068Emazatekat" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">እመ፡ ዘትካት፡ ሰከብኪ፡ ጎለ፡ አልሕምት፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">ʾƎma za-tǝkāt sakabki gola ʾalḥǝmt (Qǝne of kʷǝllǝkǝmu type)</title>
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply new/LIT7068Emazatekat.xml:9]: Yes.

File: new/LIT7068Emazatekat.xml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT7068Emazatekat" passive="BNFabb145"/>
+                </listRelation>
+            </div>
+                <div type="edition">
+                    <div type="textpart" xml:lang="gez">
+                        <note>The text is taken from <ref type="mss" corresp="BNFabb145"/>, fol. 56vc (division into verses is indicated with the four dot asterisk ።; the end of the poem is indicated with the double four dot asterisk ። ።).</note>
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply new/LIT7068Emazatekat.xml:64]: I think it is not necessary to encode it as "foreign". To make it searchable, somehow, there should be a term for this.

File: new/LIT7068Emazatekat.xml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+                        <ab>                                                  
+                            <l n="1">እመ፡ ዘትካት፡ ሰ<supplied reason="undefined" resp="DN">ከብ</supplied>ኪ፡ ጎለ፡ አልሕምት፡ ዘበክሳድኪ፡ <gap reason="illegible" unit="chars" quantity="2"/>ቅል፡ ምጽላለ፡ ኖሎት፡ ቃማ።</l> 
+                            <l n="2"><supplied reason="undefined" resp="DN">አ</supplied>ጥር፡ ብኪ፡ ተዋርዶተ፡ ቤቴል፡ <supplied reason="undefined" resp="DN">ፃማ፡</supplied></l>
```
Comment (karljonaskarlsson) [inline new/LIT7068Emazatekat.xml:67]: I read ኢ, but anything is fine, of course

File: new/LIT7068Emazatekat.xml [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+                            <l n="3">ወእመ፡ ገነት፡ ዘትካት፡ ስማ።</l>  
+                            <l n="4"><gap reason="illegible" unit="chars" quantity="2"/>ደ፡ ተልዕሎ፡ ኢናቀውማ።</l>
+                            <l n="5">ደብ<supplied reason="undefined" resp="DN">ረ፡ ጊ</supplied>ዮርጊስ፡ ከማኪ፡ ለሶዶም፡ ሀገረ፡ <supplied reason="undefined" resp="DN">ዘ</supplied>ማ።</l>
+                            <l n="6">እመኔኪ፡ መንገለ፡ ሰም<supplied reason="undefined" resp="DN">ዑ</supplied>፡ ጽራኃ፡ መላእክት፡ በዜማ።</l>
```
Comment (karljonaskarlsson) [inline new/LIT7068Emazatekat.xml:71]: እምኔኪ

File: new/LIT7068Emazatekat.xml [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+                            <l n="7">ወእ<supplied reason="undefined" resp="DN">መኔ</supplied>ሃ፡ መንገለ፡ ሰምዑ፡ ጽራኃ፡ ወ<gap reason="illegible" unit="chars" quantity="2"/>ት፡ እለ፡ በግርማ።</l>
```
Comment (karljonaskarlsson) [inline new/LIT7068Emazatekat.xml:72]: maybe supply so that it reads ወራዙት

File: new/LIT7068Emazatekat.xml [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+                            <l n="8">አኮኑ፡ ሥላሴ፡ <gap reason="illegible" unit="chars" quantity="2"/> ዝነሙ፡ ከመ፡ ዓቅምኪ፡ ወዓ<gap reason="illegible"  unit="chars" quantity="3"/>፡ ለኪ፡ መና።</l>
```
Comment (karljonaskarlsson) [inline new/LIT7068Emazatekat.xml:73]: `ወዓቅ<gap reason="illegible"`, no?

File: new/LIT7069ItyopyaAbbay.xml [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT7069ItyopyaAbbay" passive="BNFabb145"/>
+                </listRelation>
+            </div>
+            <div type="edition">
+                <div type="textpart" xml:lang="gez">
+                    <note>The text is taken from <ref type="mss" corresp="BNFabb145"/>, fol. 56vc
```
Comment (karljonaskarlsson) [inline new/LIT7069ItyopyaAbbay.xml:65]: 56vb–56vc

File: new/LIT7069ItyopyaAbbay.xml [hunk 8]
```diff
@@ -0,0 +1,83 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7069ItyopyaAbbay" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ኢትዮጵያ፡ ዓባይ፡ እምኢትዮጵያ፡ በጦመረ፡ ወይሌ፡ ወላህ፡</title>
```
Comment (karljonaskarlsson) [inline new/LIT7069ItyopyaAbbay.xml:8]: በጦማረ፡

File: new/LIT7069ItyopyaAbbay.xml [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+                    <relation name="lawd:hasAttestation" active="LIT7069ItyopyaAbbay" passive="BNFabb145"/>
+                </listRelation>
+            </div>
+            <div type="edition">
+                <div type="textpart" xml:lang="gez">
+                    <note>The text is taken from <ref type="mss" corresp="BNFabb145"/>, fol. 56vc
+                        (division into verses is indicated with the four dot asterisk ።; 
+                        the end of the poem is indicated with the double four dot asterisk ። ።).</note>
+                    <ab>                                                  
+                        <l n="1">ኢትዮጵያ፡ ዓባይ፡ እምኢትዮጵያ፡ በጦመረ፡ ወይሌ፡ ወላህ፡ ዘደመ፡ ዕጕስታር፡ ቀለሙ።</l>
```
Comment (karljonaskarlsson) [inline new/LIT7069ItyopyaAbbay.xml:69]: በጦማረ፡

File: new/LIT7069ItyopyaAbbay.xml [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+                <div type="textpart" xml:lang="gez">
+                    <note>The text is taken from <ref type="mss" corresp="BNFabb145"/>, fol. 56vc
+                        (division into verses is indicated with the four dot asterisk ።; 
+                        the end of the poem is indicated with the double four dot asterisk ። ።).</note>
+                    <ab>                                                  
+                        <l n="1">ኢትዮጵያ፡ ዓባይ፡ እምኢትዮጵያ፡ በጦመረ፡ ወይሌ፡ ወላህ፡ ዘደመ፡ ዕጕስታር፡ ቀለሙ።</l> 
+                        <l n="2">መስቆቅዋተ፡ ግብጽ፡ ጸውዒ፡ ከመ፡ ይብክያ፡ በስሙ።</l>
+                        <l n="3">ለወልደ፡ ጊዮርጊስ፡ ፍትወተ፡ ስን፡ ዓርዘ፡ ሊባኖስ፡ ዘይመስል፡ ቆሙ።</l>  
+                        <l n="4">ወወራዙትኪ፡ ኢያርምሙ።</l>
+                        <l n="5">እስመ፡ ዜና፡ ሞቱ፡ ተስምዓ፡ ቅድመ፡ ይትአምር፡ ሕማሙ።</l>
```
Comment (karljonaskarlsson) [inline new/LIT7069ItyopyaAbbay.xml:73]: I read ተሰምዓ፡ and ይትአመር፡

**Review by karljonaskarlsson — CHANGES_REQUESTED** [review]

File: new/LIT7069ItyopyaAbbay.xml (hunk 10 above)
Comment (DenisNosnitsin1970) [reply new/LIT7069ItyopyaAbbay.xml:73]: yes indeed.

File: new/LIT7069ItyopyaAbbay.xml (hunk 9 above)
Comment (DenisNosnitsin1970) [reply new/LIT7069ItyopyaAbbay.xml:69]: ok

**Comment by DenisNosnitsin1970** [comment]
Thank you for corrects! I discovered one of the qenes elsewhere.

**Review by karljonaskarlsson — APPROVED** [review]

**Pull request merged**